app/metrics.yaml

```diff
@@ -2073,6 +2073,19 @@ installation:
     notification_emails:
       - fenix-core@mozilla.com
     expires: "2020-09-01"
+  identifier:
+    send_in_pings:
+      - installation
+    type: string
+    description: |
+      The hashed and salted GAID.
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/10426
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/10446#issuecomment-624816258
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2020-05-08"
```

Comment:
I'd bump this by one more day, in case this doesn't get released until late on 5/7. We can always back it out the next day.